passes/cmds/rename.cc

```diff
@@ -252,6 +273,12 @@ struct RenamePass : public Pass {
 		log("can be used to change the random number generator seed from the default, but it\n");
 		log("must be non-zero.\n");
 		log("\n");
+		log("\n");
+		log("    rename -unescape [selection]\n");
+		log("\n");
+		log("Rename all selected public wires and cells that have to be escaped.\n");
+		log("Replaces characters with underscores or adds additional underscores and numbers.\n");
```

Comment:
I think either the option or the text should specify that it is specific to Verilog.